cts/cli/regression.error_codes.exp

```diff
@@ -126,24 +126,10 @@ Unknown - Error
 Node not found
 =#=#=#= End test: Get negative Pacemaker return code - OK (0) =#=#=#=
 * Passed: crm_error             - Get negative Pacemaker return code
-=#=#=#= Begin test: Get negative Pacemaker return code (XML) =#=#=#=
```

Comment:
I'd prefer to keep the XML test if possible, if suggested workaround for the `--` works.